images/hook/entrypoint.sh

```diff
@@ -1,58 +1,64 @@
 #!/bin/sh
 set -e
 
-echo "Assuming changeset from the environment: $RIG_CHANGESET"
+echo "---> Assuming changeset from the environment: $RIG_CHANGESET"
 # note that rig does not take explicit changeset ID
 # taking it from the environment variables
 if [ $1 = "update" ]; then
-    echo "Checking: $RIG_CHANGESET"
+    echo "---> Checking: $RIG_CHANGESET"
     if rig status $RIG_CHANGESET --retry-attempts=1 --retry-period=1s; then exit 0; fi
 
-    echo "Starting update, changeset: $RIG_CHANGESET"
+    echo "---> Starting update, changeset: $RIG_CHANGESET"
     rig cs delete --force -c cs/$RIG_CHANGESET
 
-    echo "Deleting old replication controller 'heapster'"
-    rig delete rc/heapster --resource-namespace=kube-system --force
```

Comment:
I think the update should remove both - deployment and replication controllers from previous versions.